docs/openstack/edpm_adoption.md

```diff
@@ -302,6 +267,202 @@ done
     ```
 
 * Wait for the dataplane node set to reach the Ready status:
+
     ```
     oc wait --for condition=Ready osdpns/openstack --timeout=30m
     ```
+
+## Nova compute services fast-forward upgrade from Wallaby to Antelope
```

Comment:
This section should be before creating the first OpenStackDataPlaneDeployment above as that will run a nova deployment with the wrong config (without the ffu flag) resulting in crashing nova-compute services.
```
2023-11-09 09:46:27.536 2 ERROR nova nova.exception.TooOldComputeService: Current Nova version does not support computes older than Yoga but the minimum compute service level in your cell is 56 and the oldest supported service level is 61.
```

Reply:
no, they are expected to be deployed with ffu compatibility flag set everythere

Reply:
We would have to also move nova-compute-workarounds into the main osdpns, instead of overriding it in a standalone osdpd... That would require retest FFU again

Reply:
also note that I'd prefer to keep the base osdpns clean - e.g. only referring to nova and libvirt services, w/o mentioning one-off "services" we temporarily creating to hack for FFU and to unhack it back later on

Reply:
I'll retest this again, I was observing different resuls - CP plane services start up with pre-FFU configs normally. Nova compute EDPM remains in a crash-loop, until new osdpd deployed with a pre-ffu service override.

Reply:
done https://github.com/openstack-k8s-operators/data-plane-adoption/compare/ddb74aba9c010080cd93738958cf8a56a4297731..6a5429d50247cff2290adf2dcadc2d4aedc3a49b